scripts/navit-sailfish.spec

```diff
@@ -0,0 +1,175 @@
+# $Id$
```

Comment:
What's wrong with having this under navit/contrib/sailfish ? Please do NOT duplicate this for now, as it carries release informations as well. So either move the spec file entirely (fixing my initial build script to use it as well), or leave it as it is fixing your build script.

Reply:
@metalstrolch I am in the way of first understanding how to fix the whole script. Was working on that yesterday for hours. Once I understand and fixed the issue, I could also use the spec file under navit/contrib/sailfish again. So far the only thing which worked for me, was the sed command on your file.

Reply:
@metalstrolch I now removed the second .spec file and moved the script back. Finally my custom script works. 💃